src/robot/libdocpkg/java9builder.py

```diff
@@ -0,0 +1,150 @@
+#  Copyright 2008-2015 Nokia Networks
+#  Copyright 2016-     Robot Framework Foundation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from robot.errors import DataError
+from robot import utils
```

Comment:
We've used a style to import individual utils, not the whole `utils` module. Not a big deal but consistency is nice. Being able to use, for example, `printable_name(...)` instead of `utils.printable_name(...)` also eases keeping lines below 80 chars.